ultrasonic_interpreter:
  ros__parameters:
    gazebo_topic: "/gazebo/raw/range"
    ros_topic: "/gazebo/range"

image_compressor:
  ros__parameters:
    gazebo_topic: "/gazebo/camera"
    ros_topic: "/gazebo/camera_compressed"


